### DISCUSSION TITLE
Restore Setting: Couldnt parse file error

### DISCUSSION THREAD

**Discussion by AyameNoop** [deniscerri/ytdlnis#629] [Q&A]
I tried restoring my settings from what i think is version 16 to the latest version and I got a "Couldn't Parse file" error

anything I can do or is it a lost cause?

**Comment by zaednasr** [comment]
Most likely the other items don't match from database changes. 
Modify the json file by hand. Remove the other categories and leave just the settings

**Comment by AyameNoop** [comment]
It took a few tries, but with the help of an online Json editor I did it :)

I also did notice that the Json seemed to include my download history (all 6800 videos), is there any to clean that in-app periodically?

**Comment by zaednasr** [comment]
Wdym clean? 
Old version of the app i think it just backed up everything without a choice. Now you can choose to not include it

**Comment by AyameNoop** [comment]
I mean delete my download history that is saved in app

**Comment by zaednasr** [comment]
If you don't want history then enable incognito mode
